jest.config.js

```diff
@@ -8,5 +8,7 @@ module.exports = {
   rootDir: 'test',
   modulePaths: ['<rootDir>/lib'],
   globalSetup: '<rootDir>/jest-global-setup.js',
-  globalTeardown: '<rootDir>/jest-global-teardown.js'
+  globalTeardown: '<rootDir>/jest-global-teardown.js',
+  coverageReporters: ['text', 'cobertura'],
+  collectCoverage: true
```

Comment:
```suggestion
```

I think this line can be removed as we pass `--coverage`

Reply:
Yes. Removed this.